libs/indicom.c

```diff
@@ -1647,36 +1647,52 @@ void get_equ_from_hrz(struct ln_hrz_posn *object, struct ln_lnlat_posn *observer
 
     ln_get_equ_from_hrz(&libnova_object, observer, JD, position);
 }
-
 #endif // HAVE_LIBNOVA
 
-double get_local_hour_angle(double sideral_time, double ra)
```

Comment:
is there a reason this is removed? it's not used by any drivers?

Reply:
It was not removed however, only moved.. Restored now in its previous position